source/GameInterface/Services/GameState/Patches/GameLoadedPatch.cs

```diff
@@ -5,18 +5,19 @@
 using SandBox;
 using SandBox.View.Map;
 using Serilog;
+using TaleWorlds.CampaignSystem;
 
 namespace GameInterface.Services.GameState.Patches;
 
-[HarmonyPatch(typeof(MapScreen))]
+[HarmonyPatch(typeof(Campaign))]
 internal class GameLoadedPatch
 {
     private static readonly ILogger Logger = LogManager.GetLogger<SandBoxGameManager>();
 
     [HarmonyPostfix]
-    [HarmonyPatch("OnActivate")]
+    [HarmonyPatch("OnSessionStart")]
```

Comment:
OnActivate was happening too early sometimes resulting in crashes. This hook should be more appropriate for the campaign start